### PR TITLE
ports mothplushies and flushedplush

### PR DESCRIPTION
**Pull request by spockye**
## About The Pull Request
as the name says, this PR ports a bunch of plushies from beestation, made by PigeonVerde#8826

image

image

image

## Why It's Good For The Game
you don't GET IT
WE NEED THEM

## Changelog 

:cl: PigeonVerde#8826
add: Ports mothplushies from Beestation
add: Ports flushedplush from Beestation
/:cl:

### CODE REVIEW

**Comment by Ms-Mee** [comment]
THEY TURNED MY PRECIOUS SLIME PLUSH INTO A FLUSHED EMOJI
OBLITERATE THEIR BODY AND MIND

**Comment by Ms-Mee** [comment]
Also you should probably move said plush to either before the moths or after the moth plush procs

**Comment by spockye** [comment]
good point

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -686,6 +776,11 @@
 	attack_verb = list("poked")
 	should_squeak = FALSE
 
+/obj/item/toy/plush/flushed
+	name = "flushed plushie"
+	desc = "Hgrgrhrhg cute."
+	icon_state = "flushplush"
+
 /obj/item/toy/plush/among
```
Comment (Ms-Mee) [inline code/game/objects/items/plushes.dm:784]: Amongst our flushselves

**Comment by meemofcourse** [comment]
m

**Comment by Ms-Mee** [comment]
> m

Actually, f :joker:

**Comment by Ms-Mee** [comment]
> > m
> 
> Actually, f :joker:

Call that a transplush

**Comment by PestoVerde322** [comment]
Very good Pr i approb

**Comment by thgvr** [comment]
respect

**Review by MarkSuckerberg — APPROVED** [review]
keporiplush when

**Comment by spockye** [comment]
When you get me the sprite

**Pull request merged**